authentication.md

```diff
@@ -61,18 +61,19 @@ sg | Contains the id's of the special groups to which a user belongs
 exp | Contains the expiration date when a token will expire
 
 ## Logout
-** /api/authn/logout **
+**/api/authn/logout**
 
 To logout and invalidate the JWT token, send the token in the Authorization header with the bearer scheme to the endpoint either with a GET or POST request
 
 ```
 curl -v "http://{dspace-server.url}/api/authn/logout" -H "Authorization: Bearer eyJhbG...COdbo"
 ```
 
-This invalidate the token on the server side with the result to log the user out on every device or browser.
+This invalidate the token on the server side with the result to log the user out on every device or browser. It can also be called with params **action** and **return**, required by the Shibboleth Single Logout (front channel), with the same behaviour.
 
 Return code
 - 204 No content
+- 302 Found. If a successful logout ocurs and a logout page URL is configured
```

Comment:
just a small typo here

Reply:
I fixed it, so that we can merge immediately